src/main/java/org/benetech/mathshare/model/entity/ProblemSolution.java

```diff
@@ -26,7 +25,6 @@
     @ManyToOne(cascade = CascadeType.PERSIST)
     private Problem problem;
 
-    @Lob
```

Comment:
Why is it removed? Lob allows to use very long strings

Reply:
It won't be needed there